packages/seacas/libraries/ioss/src/text_mesh/Iotm_DatabaseIO.C

```diff
@@ -643,6 +643,23 @@ namespace Iotm {
     }
   }
 
+  void DatabaseIO::update_block_omissions_from_assemblies()
+  {
+    Ioss::SerializeIO serializeIO_(this);
```

Comment:
TextMesh should not need a call to Ioss::SerializeIO since it doesn't do IO ...

Reply:
Ok, I'll remove this